swaybar/bar.c

```diff
@@ -363,7 +363,7 @@ static void handle_global(void *data, struct wl_registry *registry,
 		}
 	} else if (strcmp(interface, zwlr_layer_shell_v1_interface.name) == 0) {
 		bar->layer_shell = wl_registry_bind(
-				registry, name, &zwlr_layer_shell_v1_interface, 1);
+				registry, name, &zwlr_layer_shell_v1_interface, 2);
```

Comment:
@FlexW any idea why @ianyfan bumped this to ver 2?

Reply:
IIRC xdg-popup support in layer-shell is only available since v2?